komga/src/main/kotlin/org/gotson/komga/interfaces/api/rest/ReadListController.kt

```diff
@@ -398,6 +399,7 @@ class ReadListController(
         ZipArchiveOutputStream(responseStream).use { zipStream ->
           zipStream.setMethod(ZipArchiveOutputStream.DEFLATED)
           zipStream.setLevel(Deflater.NO_COMPRESSION)
+          zipStream.setUseZip64(Zip64Mode.Always)
```

Comment:
Could we use [`AsNeeded`](https://commons.apache.org/proper/commons-compress/javadocs/api-1.20/org/apache/commons/compress/archivers/zip/Zip64Mode.html#AsNeeded)?

Reply:
Doesn't work for this use case. From the docs:
>If the mode is AsNeeded and the entry is a compressed entry of unknown size that gets written to a non-seekable stream then change the default to Never.